### PR TITLE
OHRM5X-1213: Purge Employee and Admin Access Lang Strings

### PR DESCRIPTION
**Pull request by devishke-orange**
PR contains:
- Lang strings for Purge Employee  Screen
- Lang strings for  Administrator Access Screen
- Styling changes requested in [admin access PR](https://github.com/orangehrm/orangehrm/pull/1171)

### CODE REVIEW

File: symfony/client/src/orangehrmAuthenticationPlugin/pages/AdministratorAccess.vue [hunk 1]
```diff
@@ -154,18 +154,15 @@ export default {
 .orangehrm-admin-access {
   &-container {
     display: flex;
-    height: 100vh;
+    height: 100%;
```
Comment (devishke-orange) [inline symfony/client/src/orangehrmAuthenticationPlugin/pages/AdministratorAccess.vue:157]: https://github.com/orangehrm/orangehrm/pull/1171#discussion_r828065363

File: symfony/client/src/orangehrmAuthenticationPlugin/pages/AdministratorAccess.vue [hunk 2]
```diff
@@ -28,15 +28,16 @@
...
 
-        <oxd-text :class="noteClasses">
-          You have requested to access a critical Administrator function in
-          OrangeHRM and are required to validate your credentials below
-        </oxd-text>
+        <div :class="noteContainerClass">
+          <oxd-text tag="toast-message">
+            {{ $t('general.admin_access_note') }}
+          </oxd-text>
+        </div>
```
Comment (devishke-orange) [inline symfony/client/src/orangehrmAuthenticationPlugin/pages/AdministratorAccess.vue:40]: https://github.com/orangehrm/orangehrm/pull/1171#discussion_r828066269
Couldn't find a suitable tag for the title  (font-weight) that had both needed size and weight

File: symfony/plugins/orangehrmAuthenticationPlugin/Controller/Traits/AdministratorAccessTrait.php [hunk 3]
```diff
@@ -80,7 +82,7 @@ public function forwardToAdministratorAccess(Request $request): Response
         $urlMatcher = $this->getContainer()->get(Services::ROUTER);
         try {
             $urlMatcher->match($formattedBackUrl);
-        } catch (Exception $e) {
+        } catch (ResourceNotFoundException | NoConfigurationException | MethodNotAllowedException $e) {
```
Comment (devishke-orange) [inline symfony/plugins/orangehrmAuthenticationPlugin/Controller/Traits/AdministratorAccessTrait.php:85]: https://github.com/orangehrm/orangehrm/pull/1171#discussion_r828077983

**Review by devishke-orange — COMMENTED** [review]
I have references the comments you made in the previous PR with my changes here

File: symfony/client/src/orangehrmMaintenancePlugin/components/MaintenanceNote.vue [hunk 4]
```diff
@@ -21,9 +21,11 @@
 <template>
   <div class="orangehrm-maintenance-note">
     <div class="orangehrm-maintenance-note-header">
-      <oxd-text>Note</oxd-text>
+      <oxd-text>{{ $t('general.note') }}</oxd-text>
     </div>
-    <oxd-text>{{ noteText }}</oxd-text>
+    <oxd-text>{{
+      $t('maintenance.gdpr_note', {instanceIdentifier: instanceIdentifier})
```
Comment (RajithaKumara) [inline symfony/client/src/orangehrmMaintenancePlugin/components/MaintenanceNote.vue:27]: :ok_hand:

**Review by RajithaKumara — APPROVED** [review]

File: installer/upgrader/runTranslationTest.php [hunk 5]
```diff
@@ -33,3 +33,4 @@
 $translate->up('pim');
 $translate->up('leave');
 $translate->up('time');
+$translate->up('maintenance');
```
Comment (SharukaNP) [inline installer/upgrader/runTranslationTest.php:36]: Will have to do the same in runTranslations.php as well

Comment (SharukaNP) [reply installer/upgrader/runTranslationTest.php:36]: A good job on the translations !

Comment (devishke-orange) [reply installer/upgrader/runTranslationTest.php:36]: added

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**